src/Querier.php

```diff
@@ -255,6 +255,13 @@ public function query(Query $query)
             $solrQuery->setGroupOffset($offset);
         }
 
+        $facetSorts = $query->getFacetSorts();
+        foreach ($facetSorts as $field => $sort) {
+            $searchField = $this->getSearchField($field);
+            $facetField = $searchField->facetField();
+            $solrQuery->addParam("facet.sort.$facetField", $sort);
```

Comment:
If there is only one sort per field, you should probably use setParam instead of addParam, which avoids dealing with an array later.

Reply:
yes i've changed that